src/ripple/app/misc/AmendmentTableImpl.cpp

```diff
@@ -97,24 +99,86 @@ class AmendmentTableImpl : public AmendmentTable
     amendmentList_t getDesired();    // amendments we support, do not veto, are not enabled
 };
 
+namespace detail
+{
+using PreEnabledAmendmentsCollection = std::vector<AmendmentName>;
+
+/** preEnabledAmendments is a static collection of amendments are are enabled at build time.
+
+   Add amendments to this collection at build time to enable them on this server.
+*/
+
+PreEnabledAmendmentsCollection const preEnabledAmendments;
+}
+
 void
 AmendmentTableImpl::addInitial ()
 {
-    // For each amendment this version supports, construct the AmendmentState object by calling
-    // addKnown. Set any vetoes or defaults. A pointer to the AmendmentState can be stashed
+    detail::PreEnabledAmendmentsCollection toAdd (detail::preEnabledAmendments);
```

Comment:
Gah! When I was refactoring I introduced a bug. Fixing now (I need to add "a" to the collection in the loop). Thanks for pointing this out.